core/cloudflow-akka/src/main/resources/jdbcpersistence.conf

```diff
@@ -0,0 +1,41 @@
+akka {
+  persistence {
+    journal {
+      plugin = "jdbc-journal"
+      // Enable the line below to automatically start the journal when the actorsystem is started
+      // auto-start-journals = ["jdbc-journal"]
+    }
+    snapshot-store {
+      plugin = "jdbc-snapshot-store"
+      // Enable the line below to automatically start the snapshot-store when the actorsystem is started
+      // auto-start-snapshot-stores = ["jdbc-snapshot-store"]
+    }
+    read-journal{
+      plugin = "jdbc-read-journal"
+    }
+  }
+}
+
+jdbc-journal {
+  use-shared-db = slick
+}
+
+jdbc-snapshot-store {
+  use-shared-db = slick
+}
+
+jdbc-read-journal {
+  use-shared-db = slick
+}
+
+akka-persistence-jdbc {
+  shared-databases.slick {
+    auto-create-tables = true
```

Comment:
This setting does not exist in APJDBC. If this is to be added by Cloudflow it should not use the namespace of Akka Persistence.

Reply:
Sure